components/src/core/components/Input/CheckboxInput.vue

```diff
@@ -109,6 +110,10 @@ export default defineComponent({
       this.focused = false;
       this.$emit('blur', e);
     },
+    onClick(e: Event) {
+      this.focused = false;
+      this.$emit('click', e);    
+    },
     onChange(e: Event) {
```

Comment:
can't we do this in the same `onChange` event? since the clicking of checkbox anyway trigger `onChange` ??

Reply:
The click event is with commit 1aba5fe1a73a3193d11e1af9c08a1acb85f66729
Please refer commit message for more info